src/components/ManagerPage/AppsList.js

```diff
@@ -227,15 +268,20 @@ class AppsList extends PureComponent<Props, State> {
           </Box>
         )}
         <Text ff="Museo Sans|Regular" fontSize={6} color="dark">
-          {mode !== 'home' ? t(`manager.apps.${mode}`, { app }) : null}
+          {mode !== 'home' ? t(`manager.apps.${mode}`, { app: get(app, 'name', '') }) : null}
```

Comment:
I prefer we don't use `get(`. it is not typesafe. just use `app.name||""` (why is this needed?)

if app can be falsy, it's probably better to add the condition in the if so we never would render the wording with a falsy case 🤔